docs/src/integrations/exchange.md

```diff
@@ -488,6 +488,149 @@ You can also doublecheck whether a particular blockhash is still valid by sendin
+this amount is 0.00203928 SOL (2,039,280 lamports).
+
+#### Command Line
+To create an SPL Token account with the following properties:
+1. At a random address
+1. Associated with the given mint
+1. Owned by the funding account's keypair
+
+```
+spl-token create-account <TOKEN_MINT_ADDRESS>
+```
+
+#### Example
+```
+$ spl-token create-account AkUFCWTXb3w9nY2n6SFJvBV6VwvFUCe4KBMCcgLsa2ir
+Creating account 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+Signature: 4JsqZEPra2eDTHtHpB4FMWSfk3UgcCVmkKkP7zESZeMrKmFFkDkNd91pKP3vPVVZZPiu5XxyJwS73Vi5WsZL88D7
+```
+
+### Checking an Account's Balance
+
+#### Command Line
+```
+spl-token balance <TOKEN_ACCOUNT_ADDRESS>
+```
+
+#### Example
+```
+$ solana balance 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+0
+```
+
+### Token Transfers
+
+For SPL Token transfers to succeed, a few prerequisite conditions must be met:
+1. The recipient account must exist before the transfer is executed. As described
+in [account creation](#account-creation), SPL Token accounts are *not* explicitly
+created.
+1. Both the sender and recipient accounts must belong to the same mint.  SPL Token
+accounts can only hold one type of SPL token.
+
+#### Command Line
+```
+spl-token transfer <SENDER_ACCOUNT_PUBKEY> <AMOUNT> <RECIPIENT_ACCOUNT_PUBKEY>
+```
+
+#### Example
+```
+$ spl-token transfer 6B199xxzw3PkAm25hGJpjj3Wj3WNYNHzDAnt1tEqg5BN 1 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+Transfer 1 tokens
+  Sender: 6B199xxzw3PkAm25hGJpjj3Wj3WNYNHzDAnt1tEqg5BN
+  Recipient: 6VzWGL51jLebvnDifvcuEDec17sK6Wupi4gYhm5RzfkV
+Signature: 3R6tsog17QM8KfzbcbdP4aoMfwgo6hBggJDVy7dZPVmH2xbCWjEj31JKD53NzMrf25ChFjY7Uv2dfCDq4mGFFyAj
+```
+
+### Depositing
+Since each `(user, mint)` pair requires a separate account on chain, it is
+recommended that an exchange create batches of token accounts in advance and assign them
+to users on request. These accounts should all be owned by exchange-controlled
+keypairs.
+
+Monitoring for deposit transactions should follow the [block polling](#poll-for-blocks)
+method described above. Each new block should be scanned for successful transactions
+issuing SPL Token [Transfer](https://github.com/solana-labs/solana-program-library/blob/096d3d4da51a8f63db5160b126ebc56b26346fc8/token/program/src/instruction.rs#L92)
+or [Transfer2](https://github.com/solana-labs/solana-program-library/blob/096d3d4da51a8f63db5160b126ebc56b26346fc8/token/program/src/instruction.rs#L252)
+instructions referencing user accounts, then querying the
+[token account balance](https://docs.solana.com/apps/jsonrpc-api#gettokenaccountbalance)
+updates.
+
+[Considerations](https://github.com/solana-labs/solana/issues/12318) are being
+made to exend the `preBalance` and `postBalance` transaction status metadata
+fields to include SPL Token balance transfers.
+
+### Withdrawing
+The withdraw address a user provides must point to an initialized SPL Token account
+of the correct mint. Before executing a withdraw [transfer](#token-transfers),
```

Comment:
```suggestion
of the correct mint. Before executing a withdrawal [transfer](#token-transfers),
```

Reply:
No pass on this one since "transfer" is the noun? :swea

Reply:
https://github.com/solana-labs/solana/pull/12303#discussion_r491096121

Reply:
It's a bummer that withdrawal has multiple meanings, but it is in fact the correct word for this action. Let's just make sure not to use the phrase "going through withdrawals" in the context of exchanges 😆

Reply:
https://github.com/solana-labs/solana/pull/12303/commits/8a8c061fdf046a965e39970765978ea5335bb918 🎉